pkg/grpc/api.go

```diff
@@ -569,6 +570,29 @@ func (a *api) ExecuteStateTransaction(ctx context.Context, in *runtimev1pb.Execu
 	return &empty.Empty{}, nil
 }
 
+func (a *api) RegisterActorTimer(ctx context.Context, in *runtimev1pb.RegisterActorTimerRequest) (*empty.Empty, error) {
+	if a.actor == nil {
+		err := status.Errorf(codes.Unimplemented, messages.ErrActorRuntimeNotFound)
```

Comment:
Use InvalidArguments instead https://github.com/grpc/grpc-go/blob/master/codes/codes.go#L57? I suspect a correct actor is a required argument. Let me know if this assumption is wrong.